script/testing/util/test_case.py

```diff
@@ -0,0 +1,35 @@
+from util import constants
+from util.common import *
+
+class TestCase:
+    """Class of a test case, could be part of a test suite"""
+
+    def __init__(self, args):
+        """ Locations and misc. variable initialization """
+        # clean up the command line args
+        self.args = {k: v for k, v in args.items() if v}
+
+        # db server location
+        self.db_host = self.args.get("db_host", constants.DEFAULT_DB_HOST)
+        self.db_port = self.args.get("db_port", constants.DEFAULT_DB_PORT)
```

Comment:
Test case class needs the db_host and db_port information to build the DB URL for the JDBC connection.

Reply:
Could we build the DB URL in the test_server.py? It seems like something that would be constant over the course of all the tests rather than changing per test.